core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -115,6 +117,14 @@ public void run() {
             config.getMetricsHostAllowList());
 
     final MetricsSystem metricsSystem = metricsEndpoint.getMetricsSystem();
+
+    final SlashingProtection slashingProtection;
```

Comment:
may I suggest Optional<SlashingProtection> instead of null? or may be a createNoOpSlashingProtection?

Reply:
nevermind ... I see that you have eventually used Optional. So its OK for now.

Reply:
Rather we didn't have to pass null around